src/cmds/logout.ts

```diff
@@ -0,0 +1,21 @@
+import netrc from 'netrc'
+import { logSuccess } from '../logger'
+import { getNetrcEntryId } from '../utils/auth'
+
+const CMD = 'logout'
+
+function addLogoutCmd(program) {
+    program.command(CMD).action(logout)
+}
+
+/**
+ * Logout of Spec.
+ */
+async function logout() {
+    const entries = netrc()
```

Comment:
Let's break this logic out into a function called `deleteSession()` inside the `utils/auth.ts` and wrap it in a try/catch with a very similar structure to `persistSession`. Then from here, we can do something like:
```typescript
const { error } = deleteSession()
if (error) {
    logFailure(error)
    return
}
logSuccess('Successfully logged out.')